core/src/main/resources/hudson/model/Job/index.jelly

```diff
@@ -40,6 +40,9 @@ THE SOFTWARE.
             <l:breakable value="${it.displayName}"/>
           </h1>
         </div>
+        <div class="jenkins-app-bar__controls">
+          <t:editDescriptionButton permission="${it.UPDATE}"/>
```

Comment:
There's no `Job.UPDATE`, is there? Is there any reason you went for a change?

Reply:
thanks fixed